site/src/pages/LoginPage/SignInForm.tsx

```diff
@@ -91,7 +91,7 @@ export const SignInForm: FC<React.PropsWithChildren<SignInFormProps>> = ({
         </div>
       )}
 
-      {Boolean(info) && Boolean(error) && (
+      {Boolean(info) && (
```

Comment:
@aslilac just want to double-check we don't need this error check here.

Reply:
yeah, this is probably fine